js_modules/dagit/packages/core/src/launchpad/LaunchpadSetupRoot.tsx

```diff
@@ -51,7 +51,10 @@ const LaunchpadSetupAllowedRoot: React.FC<Props> = (props) => {
       }
       if (queryString.solidSelection instanceof Array) {
         newSession.solidSelection = queryString.solidSelection as string[];
-      } else if (typeof queryString.solidSelection === 'string') {
+      } else if (
+        typeof queryString.solidSelection === 'string' &&
+        queryString.solidSelection.length > 0
```

Comment:
nit: 0-length strings are false-y, so you could also do something like `typeof queryString.solidSelection === 'string' && queryString.solidSelection`